com.unity.netcode.gameobjects/Tests/Runtime/Serialization/NetworkObjectReferenceTests.cs

```diff
@@ -0,0 +1,256 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Object = UnityEngine.Object;
+
+namespace Unity.Netcode.RuntimeTests.Serialization
```

Comment:
let's not put tests under specific/separate namespaces and keep using the common top-level one?
```suggestion
namespace Unity.Netcode.RuntimeTests
```

Reply:
ugh, we're going to play "find where my tests are" game with foldout menus instead of a flat hierarchy :P
it's fine though, it's kind of a personal preference — I just wanted to follow the same "per-asmdef namespaces" here too but can live without it.